Makefile.erb

```diff
@@ -0,0 +1,205 @@
+SHELL := /bin/bash
+
+# Some useful variables...
+
+ERROR   := "   \033[41;1m error \033[0m "
+INFO    := "    \033[34;1m info \033[0m "
+OK      := "      \033[32;1m ok \033[0m "
+WARNING := " \033[33;1m warning \033[0m "
+
+# Task: help | show this help message.
+# {
+.PHONY: help
+help:
+	@echo
+	@echo -e '\033[33mUsage:\033[0m'
+	@echo '  make <target> [--] [<command>...] [options]'
+	@echo
+	@echo -e '\033[33mTargets:\033[0m'
+	@egrep '^# Task: ([a-z-]+) \| (.+)$$' Makefile | awk -F '[:|]' '{print " \033[32m"$$2"\033[0m" "|" $$3}' | column -t -s '|'
+	@echo
+# }
+
+
+# Task: build | create the application docker image.
+# {
+.PHONY: build
+build:
+	@bash -c 'bundle exec rake docker:build'
+# }
+
+
+# Task: setup | initial project setup and configuration.
+# {
+.PHONY: setup
+setup:
```

Comment:
this could for example be a project rake task called setup, and need only run `rake setup` instead of `make setup`